setup.cfg

```diff
@@ -29,6 +29,7 @@ install_requires =
     dimod==0.12.3
     dwave-cloud-client==0.10.4
     dwave-greedy==0.3.0
+    dwave-gate==0.1.0
```

Comment:
I don't think we want to add it to the "main packages" yet?

Reply:
Ah, good point.